### ISSUE TITLE
[BUG] Element screenshot displaying other covering elements

### ISSUE THREAD

**Issue by nboisteault** [microsoft/playwright#15572]
**Context:**
- Playwright Version: 1.23.2
- Operating System: Linux
- Node.js version: 16.16.0
- Browser: All
- Extra: 
## System:
 - OS: Linux 5.15 Ubuntu 22.04 LTS 22.04 LTS (Jammy Jellyfish)
 - Memory: 1.46 GB / 7.55 GB
 - Container: Yes
## Binaries:
 - Node: 16.16.0 - /usr/bin/node
 - npm: 8.11.0 - /usr/bin/npm
## Languages:
 - Bash: 5.1.16 - /usr/bin/bash

**Code Snippet**

```javascript
const { test, expect } = require('@playwright/test');

test.beforeEach(async ({ page }) => {
  await page.goto('https://demo.lizmap.com/lizmap/index.php/view/map/?repository=misc1&project=qgis_info');
});


test.describe('Startup', () => {
  test.only('screenshot', async ({ page }) => {
    await page.locator('#map').screenshot({path: 'screenshot.png'});
  });
});
```

**Describe the bug**

When a screenshot is made of the map element (`#map`), other elements covering it are visible while in [documentation](https://playwright.dev/docs/api/class-locator#locator-screenshot) it is written : 

> If the element is covered by other elements, it will not be actually visible on the screenshot

In the result here elements as the modal window, navigation bar, logo at the bottom etc. are visible.

![screenshot](https://user-images.githubusercontent.com/2145040/178536286-4ecb8323-1595-4474-97b7-1d848201252b.png)

**Comment by yury-s** [comment]
As mentioned above it is working as intended, the parts of the #map that are occluded by other elements are not on the screenshot.

**Comment by nboisteault** [comment]
> From my understanding this works as intended as the doc means If the element is covered by other elements, it _(the element)_ will not be actually visible on the screenshot
> 
> Maybe the documentation is unclear ? Do you think we can rephrase it better ?

I would rephrase:

> Parts of the element covered by other elements will no be actually visible on the screenshot.
